Travel Scheduler/Helper Files/PlacesToVisitCollectionView.m

```diff
@@ -0,0 +1,15 @@
+//
+//  PlacesToVisitCollectionView.m
+//  Travel Scheduler
+//
+//  Created by gilemos on 7/24/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "PlacesToVisitCollectionView.h"
+
+@implementation PlacesToVisitCollectionView
```

Comment:
Nice implementation of PlacesToVisitCollectionView. I left this in the home view controller file which was a bad way to have done it